indico/modules/events/contributions/__init__.py

```diff
@@ -106,7 +106,9 @@ def _extend_event_menu(sender, **kwargs):
 
     def _visible_my_contributions(event):
         published = contribution_settings.get(event, 'published')
-        return session.user and published and has_contributions_with_user_as_submitter(event, session.user)
+        if not session.user:
+            return False
+        return has_contributions_with_user_as_submitter(event, session.user) or published
```

Comment:
I don't think `has_contributions_with_user_as_submitter` is correct - AFAIK we allow any author to submit editables (even though I think there's an issue to change this?)

It should be possible to create a util that has similar logic to check for the whole event, just like `is_user_associated` does for a single contribution

Reply:
This about showing "My Contributions" menu entry to users who are contribution authors though, right?